### PR TITLE
LRSD-7363 Style inconsistencies in Learn

### PR DESCRIPTION
**Pull request by GabrielFernandoS**

### CODE REVIEW

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -888,6 +892,12 @@ pre[class*='language-'] {
 	}
 }
 
+.learn-category div {
+	color: #282934;
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:896]: spacing, and use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 2]
```diff
@@ -1087,6 +1097,27 @@ pre[class*='language-'] {
...
 }
 
+.panel-title {
+	color: #282934;
+	font-size: 1.125rem;
+	text-transform: none;
+}
+
+.custom-control-label .custom-control-label-text {
+	color: #282934 !important;
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1107]: use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 3]
```diff
@@ -1087,6 +1097,27 @@ pre[class*='language-'] {
 	}
 }
 
+.panel-title {
+	color: #282934;
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1101]: use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 4]
```diff
@@ -1087,6 +1097,27 @@ pre[class*='language-'] {
...
+	text-transform: none;
+}
+
+.custom-control-label .custom-control-label-text {
+	color: #282934 !important;
+	line-height: 16px;
+}
+
+.custom-control-label::before {
+	border-color: #282934 !important;
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1112]: use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 5]
```diff
@@ -1087,6 +1097,27 @@ pre[class*='language-'] {
...
+}
+
+.treeview .btn {
+	color: #282934 !important;
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1116]: use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 6]
```diff
@@ -1440,7 +1471,14 @@ pre[class*='language-'] {
 				}
 			}
 		}
-	}
+	} 
+}
+
+.input-group .search-bar-keywords-input::placeholder {
+	line-height: 24px !important;
+	color: #6C6C75 !important;
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1479]: use variable colors from/liferay-portal/workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/style-books/liferay-learn/frontend-tokens-values.json

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by rtienpal** [comment]
ci:test:sf

**Pull request closed without merging**